### DISCUSSION TITLE
How to install rrdtool for python on windows

### DISCUSSION THREAD

**Discussion by music-fan** [oetiker/rrdtool-1.x#1201] [Q&A]
Hi, I'm using rrdtool as python module on my small raspberry with Debian since many years but I was never able to use on my windows PC (in python) even there is (now) a binary versions for windows. Can anyone make a How-To-Install for using rrdtool with python (3.x) on windows (10, x64)? Thanks in advance!

**Comment by de2Zotjes** [comment]
Any interest in a pure python rrd reading library? I built this for my own use, but I am wondering if anyone else might want to use it.
